static/styles.js

```diff
@@ -1,212 +1,183 @@
-    // get all id tags
-    let body = document.getElementsByTagName("body")[0]; 
-    let chat = document.getElementById("chat");
-    let message = document.getElementById("message");
-    let chatbox = document.getElementById("chatbox");
-    let sides = document.getElementById("sides");
-    let topleft = document.getElementById("topleft");
-    let send = document.getElementById("send");
-    let sidebar = document.getElementById("activenav");
-    let sidenav = document.getElementsByClassName("sidenav")[0]
-    let snav_text = sidenav.getElementsByTagName("a")
-    let snav_iter = snav_text.length;
-    // then set the color to what it is in the css document
-    body.style.backgroundColor = "#000000";
-    chat.style.color = "#228e3d";
-    message.style.color = "#000000";
-    chatbox.style.backgroundColor = "#181616";
-    sides.style.backgroundColor = "#121212";
-    sidebar.style.backgroundColor = "#171717"
-    sidebar.style.color = "#ffffff"
-    topleft.style.backgroundColor = "#121212";
-    topleft.style.color = "#696969";
-    send.style.backgroundColor = "#192080";
-    send.style.color = "#ffffff";
-    sidenav.style.backgroundColor = "#111";
-    // for loop to cycle thru links in sidebar
-    for (let i = 0; i < snav_iter; i++) {
-        snav_text[i].style.color = "#818181";
-    }
-}
-
-function setDevStyle() {
-    // get all id tags
-    let body = document.getElementsByTagName("body")[0]; 
-    let chat = document.getElementById("chat");
-    let message = document.getElementById("message");
-    let chatbox = document.getElementById("chatbox");
-    let sides = document.getElementById("sides");
-    let topleft = document.getElementById("topleft");
-    let send = document.getElementById("send");
-    let sidebar = document.getElementById("activenav");
-    let sidenav = document.getElementsByClassName("sidenav")[0];
-    let snav_text = sidenav.getElementsByTagName("a");
-    let snav_iter = snav_text.length;
-    // then set the color to what it is in the css document
-    body.style.backgroundColor = "#000000";
-    chat.style.color = "#18691f";
-    message.style.color = "#000000";
-    chatbox.style.backgroundColor = "#0d0d0d";
-    sides.style.backgroundColor = "#080808";
-    sidebar.style.backgroundColor ="#080808";
-    topleft.style.backgroundColor = "#0d0d0d";
-    topleft.style.color = "#ffffff";
-    send.style.backgroundColor = "#006600";
-    send.style.color = "#ffffff";
-    sidenav.style.backgroundColor = "#0f0f0f";
-    // for loop to cycle thru links in sidebar
-    for (let i = 0; i < snav_iter; i++) {
-        snav_text[i].style.color = "#8a4e11";
-    }
-}
\ No newline at end of file
+const winterTheme = [
+    "#AED9FF",
+    "#d64304",
+    "#000000",
+    "#FF9933",
+    "#121212",
+    "#FF9933",
+    "#f5f2f2",   // Sidebar Background Color
+    "#3b3b3b",   // Sidebar Border Color
+    "#ffffff",   // Sidebar Text Color
+    "#FF9933",
+    "#d64304",
+    "#ffffff",
+    "#111",
```

Comment:
line 179 should be expanded out to `#111111`

Reply:
Completed